content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -1,59 +1,66 @@
 ---
 linktitle: Creating Projects Using the CLI
 title: Creating Projects Using the Command Line Interface
-description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the CLI.
+description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the command line interface (CLI).
 weight: 200
 toc: true
 ---
 
-The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the command line interface (CLI).
+The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the `o3de` command line interface (CLI) tool. The O3DE CLI is a Python script used for project configuration. Invoke it from the `scripts` directory in your O3DE install. For a complete guide to O3DE CLI commands, refer to the [Project Configuration CLI Reference](/docs/user-guide/project-config/cli-reference).
```

Comment:
Just added some explanatory text for the `o3de` command line interface.

@paucom I'd like your confirmation on the correctness of my capitalization of "o3de" throughout this paragraph. I used lowercase and code formatting when I referred to the tool by its filename (`o3de`), and uppercase when referring to the abbreviated proper name (O3DE CLI). Do you agree with this formatting? Should I use bold type on the first appearance of **O3DE CLI**?